### PR TITLE
NodeGraph: Schema for panel options

### PR DESCRIPTION
**Pull request by aocenas**
Make a schema for PanelOption and use the generated types.

### CODE REVIEW

File: public/app/plugins/panel/nodeGraph/panelcfg.cue [hunk 1]
```diff
@@ -0,0 +1,55 @@
+// Copyright 2022 Grafana Labs
```
Comment (Clarity-89) [inline public/app/plugins/panel/nodeGraph/panelcfg.cue:1]:
```suggestion
// Copyright 2023 Grafana Labs
```

File: public/app/plugins/panel/nodeGraph/panelcfg.cue [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+						}
+						EdgeOptions: {
+							// Unit for the main stat to override what ever is set in the data frame.
+							mainStatUnit?: string
+							// Unit for the secondary stat to override what ever is set in the data frame.
+							secondaryStatUnit?: string
+						}
+						PanelOptions: {
+							nodes?: NodeOptions
+							edges?: EdgeOptions
```
Comment (Clarity-89) [inline public/app/plugins/panel/nodeGraph/panelcfg.cue:48]: Do these two fields need some kind of comments? The description in the generated doc is empty.

**Review by Clarity-89 — APPROVED** [review]
Great work! 🎉  Left a few minor comments.

File: public/app/plugins/panel/nodeGraph/panelcfg.cue (hunk 2 above)
Comment (aocenas) [reply public/app/plugins/panel/nodeGraph/panelcfg.cue:48]: Honestly not sure if `// Options for nodes // Options for edges` give any more valuable information and not sure what else to write, it's just so the options are in separate sections.

**Pull request merged**